snuba/utils/streams/topics.py

```diff
@@ -25,12 +25,14 @@ class Topic(Enum):
     SUBSCRIPTION_RESULTS_SESSIONS = "sessions-subscription-results"
     SUBSCRIPTION_RESULTS_METRICS = "metrics-subscription-results"
     QUERYLOG = "snuba-queries"
+    PROFILES = "processed-profiles"
 
 
 def get_topic_creation_config(topic: Topic) -> Mapping[str, str]:
     config = {
         Topic.EVENTS: {"message.timestamp.type": "LogAppendTime"},
         Topic.METRICS: {"message.timestamp.type": "LogAppendTime"},
+        Topic.PROFILES: {"message.timestamp.type": "LogAppendTime"},
```

Comment:
If I understand correctly, subscriptions/alerting is useful to send an email to a user saying something happened, right? I don't think we'll want to do that on individual profiles, I think it's something we'd want to do on aggregations (like, the 95th percentile timing of that function went over X ms when looking at high end phones on iOS). That's why I left it out for now because I don't think that's how I'd do that right now?

Reply:
Yees subscriptions would allow you to run those kinds of aggregate queries over your project data on a regular interval. Not important now, I was just curious, subscription topics are the ones where we need LogAppendTime.